.github/workflows/ci.yml

```
@@ -193,6 +204,118 @@ jobs:
+
+    - name: Post bundle-size comment to PR
+      if: github.event_name == 'pull_request'
+      uses: peter-evans/create-or-update-comment@v4
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        body-path: ${{ runner.temp }}/bundle-stats.md
+        # body-includes matches the HTML comment tag emitted by
+        # parse-bundle-stats.mjs so subsequent pushes update the same
+        # comment instead of appending a fresh one.
+        edit-mode: replace
+        body-includes: "<!-- bundle-size-report -->"
```

actionlint (1.7.12)

[error] input "body-includes" is not defined in action "peter-evans/create-or-update-comment@v4". available inputs are "append-separator", "body", "body-file", "body-path", "comment-id", "edit-mode", "issue-number", "reactions", "reactions-edit-mode", "repository", "token"

(action)